projects/igniteui-angular/src/lib/grids/pinning/column-pinning.component.html

```diff
@@ -21,3 +21,8 @@ <h4 class="igx-column-hiding__header-title" *ngIf="title">{{ title }}</h4>
         {{ columnItem.name }}
     </igx-checkbox>
 </div>
+
+<div class="igx-column-hiding__buttons">
```

Comment:
Can we call the class something more generic as it's used on two places now? `igx-toolbar-dropdown_buttons" or something similar?

Reply:
If we will rename this class, I think that we'd better rename all the classes that are used by both column-hiding and column-pinning templates:

igx-column-hiding__header
igx-column-hiding__header-title
igx-column-hiding__header-input
igx-column-hiding__columns
igx-column-hiding__columns-item

Reply:
@simeonoff , what do you think?